src/main/java/glowredman/txloader/JarHandler.java

```diff
@@ -19,8 +19,24 @@ class JarHandler {
 
     static void indexJars() {
         String userHome = System.getProperty("user.home");
-        txloaderCache = Paths.get(userHome, "txloader");
-
+        String system = System.getProperty("os.name").toLowerCase();
+        if (system.contains("win")) {
+            String temp = System.getenv("TEMP");
+            if (temp == null) {
+                txloaderCache = Paths.get(userHome, "AppData", "Local", "Temp", "txloader");
```

Comment:
appdata and temp folders can be in a different locations on windows, you need to check the environment variables for it

Reply:
Now %LOCALAPPDATA% is checked